v2/samples/dbformysql/v1api/v1_user.yaml

```diff
@@ -21,6 +21,8 @@ spec:
       - SELECT
   localUser:
     serverAdminUsername: admin
+    # serverAdminPassword is optional. If NOT specified, the operator uses its identity to log into the server.
+    # The operator can only successfully log into the server if its identity is the Administrator of the server.
```

Comment:
Can the Administrator of the server be a group?

Reply:
Yes! I've added some clarifying documentation about how exactly to log in when the admin is a member of the admin group.

Specifically:
```
    # If the administrator is a group, the ServerAdminUsername should be the group name, not the actual username of the
    # identity to log in with. For example if the administrator group is "admin-group" and identity "my-identity" is
    # a member of that group, the ServerAdminUsername should be "admin-group"
```

That same documentation now exists in the CRD types too (Local and AAD flavors)